pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -41,7 +41,7 @@ func (r *Reconciler) ReconcileAPIUrl(ctx context.Context, dynakube *dynatracev1b
 	return nil
 }
 
-func (r *Reconciler) ReconcileCommunicationHosts(ctx context.Context, dynakube *dynatracev1beta1.DynaKube) error {
+func (r *Reconciler) ReconcileCMCommunicationHosts(ctx context.Context, dynakube *dynatracev1beta1.DynaKube) error {
```

Comment:
It's not clear what's `CM`?

Reply:
CM == CodeModules (but I mentioned that the names are still not the best 😅 )

Reply:
done